c_src/h3.c

```diff
@@ -690,7 +798,27 @@ static ErlNifFunc nif_funcs[] =
      {"max_k_ring_size", 1, erl_max_k_ring_size, 0},
      {"indices_are_neighbors", 2, erl_indices_are_neighbors, 0},
      {"get_unidirectional_edge", 2, erl_get_unidirectional_edge, 0},
-     {"grid_distance", 2, erl_grid_distance, 0}};
+     {"grid_distance", 2, erl_grid_distance, 0},
+     {"get_origin_from_unidirectional_edge",
+      1,
```

Comment:
Could you put each of these on one line, like the others?

Reply:
> On the 80 column question, you mean for clang-format?
This is due to the formatting here :(

I will be following up with a PR to split some tests as well.